src/annotations/AnnotationParser.php

```diff
@@ -91,7 +101,7 @@ public function parse($source, $path)
 
             switch ($state) {
                 case self::SCAN:
-                    if ($type == T_CLASS) {
+                    if ($type == T_CLASS || $type == T_TRAIT) {
```

Comment:
Since we support trait scanning now you need to remove code in `AnnotationManager::getClassAnnotations` that forbids that.